src/iexamine.cpp

```diff
@@ -1473,6 +1473,39 @@ void iexamine::fault( player &, const tripoint & )
               "shape, but with long, twisted, distended limbs." ) );
 }
 
+/**
+ * Display popup message pulled from the object's message property
+ */
+void iexamine::notify( player &p, const tripoint &pos )
+{
+    std::string message = g->m.has_furn( pos ) ?
+                          g->m.furn( pos ).obj().message :
+                          g->m.ter( pos ).obj().message;
+    if( !message.empty() ) {
+        popup( _( message ) );
+    }
+}
+
+/**
+* Transform the examined object into the object specified by its transforms_into property. If the new object has a message property,
+* it is displayed as if the notify examine_action was used.
+*/
+void iexamine::transform( player &p, const tripoint &pos )
```

Comment:
The `p` parameter isn't used. Same in `notify`.
If you aren't using a parameter, remove its name, but keep its type.
```suggestion
void iexamine::transform( player &, const tripoint &pos )
```